.github/docker-images/swift-5-centos-x64/Dockerfile

```diff
@@ -5,39 +5,15 @@ FROM swift:5.4-centos7
 # Install prereqs
 # any prereqs that appear to be missing are installed on base swift image i.e. tar, git
 ###############################################################################
-RUN yum -y update \
```

Comment:
Performing this update on centos was breaking glibc's headers, it appears to actually downgrade them.